scout/adapter/mongo/case.py

```diff
@@ -102,24 +103,20 @@ def _set_similar_phenotype_query(
                 hpo_terms.append(term.get("phenotype_id"))
             similar_cases = self.cases_by_phenotype(hpo_terms, institute_id, case_obj["_id"])
         else:  # similar HPO terms
-            hpo_terms = list(query_term.replace(" ", "").split(","))
+            hpo_terms = list(query_value.replace(" ", "").split(","))
```

Comment:
Just renamed he variable to be more clear

Reply:
💯

Reply:
Query term was more logical when the term was in there, and not in a separate field I guess.. 😁